soh/src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -349,7 +349,17 @@ void EnDoor_Draw(Actor* thisx, PlayState* play) {
             }
         }
         if (this->lockTimer != 0) {
+            if (CVarGetInteger("gShowDoorLocksOnBothSides", 0)) {
+                Matrix_Push();
+            }
             Actor_DrawDoorLock(play, this->lockTimer, DOORLOCK_NORMAL);
+            if (CVarGetInteger("gShowDoorLocksOnBothSides", 0)) {
+                Matrix_Pop();
+                Matrix_Push();
+                Matrix_RotateZYX(0, 0x8000, 0, MTXMODE_APPLY);
+                Actor_DrawDoorLock(play, this->lockTimer, DOORLOCK_NORMAL);
+                Matrix_Pop();
```

Comment:
Is this second set of mtx push/pop necessary? I understand the first set is to prevent mtx modifications from the first lock applying to the second lock, but there is nothing being rendered after the second lock.

Reply:
You're right they can be dropped 👍

Reply:
```suggestion
                Matrix_RotateZYX(0, 0x8000, 0, MTXMODE_APPLY);
                Actor_DrawDoorLock(play, this->lockTimer, DOORLOCK_NORMAL);
```